webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.scss

```diff
@@ -11,13 +12,18 @@
   }
 
   .ReactPasswordStrength-strength-desc {
-    padding: 6px 10px;
+    padding: 2px 10px;
     right: -62.5%;
     font-size: 12px;
     width: 60%;
     text-align: left;
     font-style: normal;
     font-family: $font-family-base;
+
+    @media (max-width: $screen-sm-max) {
+      right: 45%;
```

Comment:
I played with the css and found out that it is better to just use `right: auto;`.
Can you try it?